### PR TITLE
Classify genotypes plates 3 3p

### PR DESCRIPTION
**Pull request by MattsonCam**
In this pr, two logistic regression models are trained on both the shuffled and unshuffled plate (3 and 3 prime) combined data to classify nf1 genotypes. Please let me know if you have any questions or suggestions.

### CODE REVIEW

File: 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+# ## Use only common columns
+
+# In[7]:
+
+
+# Set plate column:
+plate3df["Metadata_plate"] = "3"
+plate3pdf["Metadata_plate"] = "3p"
+
+common_columns = list(plate3df.columns.intersection(plate3pdf.columns))
```
Comment (MikeLippincott) [inline 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py]:
```suggestion
common_columns = (plate3df.columns.intersection(plate3pdf.columns).to_list()
```
consider changing this line

File: 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+# # Train Models
+
+# ## Define model variables
+
+# In[9]:
+
+
+# Define the hyperparameter search space
+param_dist = {
```
Comment (MikeLippincott) [inline 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py:138]: consider adding other hyperparameters to search too like:
`max_iter`,`multi_class` ovr or ova

**Review by MikeLippincott — APPROVED** [review]
Solid PR. I left some small comments that you can address if you think it is worth your while. LGTM, ready to merge!

File: 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py (hunk 1 above)
Comment (MattsonCam) [reply 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py]: Changed!

File: 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py (hunk 2 above)
Comment (MattsonCam) [reply 1.train_models/log_reg_plates_cp_fs_data/plates_3_3p/nbconverted/plates_3_3p_classify_genotype.py:138]: Sounds good, I'll keep this in mind for later

**Comment by MattsonCam** [comment]
Thanks Mike, merging now!

**Pull request merged**